deploy/addons/olm/olm.yaml.tmpl

```diff
@@ -308,7 +310,7 @@ spec:
                 - --global-namespace
                 - olm
                 image: {{.CustomRegistries.OLM  | default .ImageRepository | default .Registries.OLM }}{{.Images.OLM}}
-                imagePullPolicy: IfNotPresent
+                imagePullPolicy: Always
```

Comment:
is this change related to bumping the version ?

Reply:
yes